.github/workflows/publish.yml

```diff
@@ -0,0 +1,16 @@
+name: Docker build image and publish
+
+on:
+   push:
+     branches: [ master ]
+     
+jobs:
+  build-image:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build the Docker image and publish it
+      run: |
+        docker build . --file embedded/Dockerfile --tag swarmus/embedded:latest
+        docker login --username glessard --password ${{ secrets.DOCKER_TOKEN }}
```

Comment:
We might want to have a mode generic user

Reply:
I'd have to remove Xavier or Charles from the Docker Organization. Who will be the sacrificial lamb?

Reply:
You can remove me, I don't mind